### DISCUSSION TITLE
Linear integration

### DISCUSSION THREAD

**Discussion by innocenzi** [spatie/flareapp.io-roadmap#80] [Ideas]
Hey, it would be awesome if we could create an issue from Flare to Linear.

**Comment by pedroborges** [comment]
Coming from Sentry so far the only feature we are missing in Flare is the Linear integration.

**Comment by Konafets** [comment]
Another vote for FlareApp integration.

**Comment by freekmurze** [comment]
We're currently implementing this. ETA Q2 2024 👍

**Comment by WouterBrouwers** [comment]
🎉  Our Linear integration is now live: 
https://flareapp.io/blog/introducing-linear-integration
